pkgs/servers/dns/bind/default.nix

```
@@ -22,6 +22,11 @@ stdenv.mkDerivation rec {
   patches = [
     ./dont-keep-configure-flags.patch
     ./remove-mkdir-var.patch
+    # Fix cross-compilation (will be included in next release)
```

```suggestion
    # Fix cross-compilation (will be included in next release after 9.16.8)
```

I like it when you mention the release explicitly just to be sure you know when to remove it.